### DISCUSSION TITLE
Récupérer nom de d'une scène depuis une autre scène

### DISCUSSION THREAD

**Discussion by Synfactory** [witsa/synapps#711] [Q&A] [answered]
Hello,
je souhaiterai récupérer le nom d'une scene et pas sa clé depuis une autre scène pour rendre dynamique un texte.
Dans ma scene principale j'ai un screen qui va chercher cette scène.

J'ai fait ça sur mon texte :
![image](https://github.com/witsa/synapps/assets/165296029/531c729e-30e7-4517-9174-2f35d33d6b1e)
Mais ça affiche la scenekey et non pas le nom de la scene. Y'aurait une autre manip?
Merci

**Comment by WITNicolas** [comment, marked as answer]
Bonjour,
Effectivement il y a une subtilité. <s>Dans le chemin, il faut mettre : scenekey.name
scenekey tout seul ne suffit pas.</s>

Edit (merci manu) : 
Bien vu j'ai répondu trop vite ... : `displayedScene.name`

**Reply by manufitoussiwit** [reply]
Salut, il me semble que c'est plutôt `displayedScene.name`.

`displayedScene` c'est le champ d'information qui permet d'obtenir l'objet scène qui est affichée. Après, on peut accéder à tous les champs de la scène comme si on faisait une liaison sur une scène directement (C'est une méthode pour découvrir ces champs d'ailleurs). Ici, on accède au nom : `name`.

**Reply by Synfactory** [reply]
Super, merci beaucoup. !
